src/data/faq.json

```diff
@@ -1501,8 +1526,7 @@
                                 "anchor": "mask_rules",
                                 "active": false,
                                 "textblock": [
-                                    "<b>Updated, October 21, 2022</b>",
-                                    "<div class='update'>As of version 2.27, the Corona-Warn-App (CWA) displays the current mask status as a dedicated symbol in the \"Certificates\" area if any federal states define their own more extensive mask obligations.<br>According to the fall/winter plan of the federal government this is divided into the \"federal legislations\" and \"supplemental measures of the federal states\" (cf. sources below).<br><b>Currently, there are no additional measures of the states.</b><hr/>",
+                                    "<div class='update'>The information in this FAQ entry is related to features or describes features of the app that are no longer available as of May 1, 2023, and is therefore no longer up-to-date. For more information, click here: <a href='#ramp_down' target='_blank'>What is the background for the end of the development of the CWA?</a><hr />As of version 2.27, the Corona-Warn-App (CWA) displays the current mask status as a dedicated symbol in the \"Certificates\" area if any federal states define their own more extensive mask obligations.<br>According to the fall/winter plan of the federal government this is divided into the \"federal legislations\" and \"supplemental measures of the federal states\" (cf. sources below).<br><b>Currently, there are no additional measures of the states.</b><hr/>",

```

Comment:
"fall/winter plan" this is dated and it is unclear what year this belongs to

Reply:
Added the year in https://github.com/corona-warn-app/cwa-website/pull/3474/commits/3420f6ef18b4aab32751f66db3cda39b99703297, thanks!